### PR TITLE
Differentiate search and select styles

### PR DESCRIPTION
**Pull request by xiajenny**
Screen Shot 2022-03-06 at 6 34 44 PM
Screen Shot 2022-03-06 at 6 34 52 PM

### CODE REVIEW

File: frontend/src/components/MapRenderer.ts [hunk 1]
```diff
@@ -1,42 +1,86 @@
...
 
-const BLUE = '#8FB1BB';
-const DARK_BLUE = '#1A85A7';
-const ORANGE = '#F29D49';
-const BEIGE = '#B4A88C';
-const LIGHT_BEIGE = '#D7CFBE';
+const SEARCH_ITEM_COLOUR = '#1A85A7'; // teal
+const SELECTED_ITEM_COLOUR = '#E28D38'; // orange
+
+// const LIGHT_BEIGE = '#D7CFBE';
```
Comment (MickeyDang) [inline frontend/src/components/MapRenderer.ts:13]: Delete stray comment?

File: frontend/src/components/MapRenderer.ts [hunk 2]
```diff
@@ -1,42 +1,86 @@
...
 const navigation = new mapboxgl.NavigationControl({ showCompass: false });
 
 let sourceLoaded = false;
 
+function seperateProjectSources(
```
Comment (MickeyDang) [inline frontend/src/components/MapRenderer.ts:18]: Thanks for making this into a function :)

File: frontend/src/components/MapRenderer.ts [hunk 3]
```diff
@@ -1,42 +1,86 @@
...
   isSearchMode: boolean,
   annotations: Annotations,
+  selectedProject: Project,
   map: mapboxgl.Map,
 ) {
   if (sourceLoaded) {
-    const reportSource: mapboxgl.GeoJSONSource = map.getSource('reports') as mapboxgl.GeoJSONSource;
-    const relationshipSource: mapboxgl.GeoJSONSource = map.getSource('relationships') as mapboxgl.GeoJSONSource;
+    const data = seperateProjectSources(reports, relationships, selectedProject);
+    console.log(data);
```
Comment (MickeyDang) [inline frontend/src/components/MapRenderer.ts:59]: Remove console.log

**Review by MickeyDang — APPROVED** [review]
Should annotation polygons be beige like the Figma?

Can be in a followup PR or this one?

**Comment by MickeyDang** [comment]
Referring to annotation example

Screen Shot 2022-03-06 at 6 43 45 PM

File: frontend/src/components/MapRenderer.ts (hunk 1 above)
Comment (xiajenny) [reply frontend/src/components/MapRenderer.ts:13]: gonna make annotations beige in followup

Comment (MickeyDang) [reply frontend/src/components/MapRenderer.ts:13]: Sounds good! Thanks for making small PRs thats actually very much appreciated :)

**Pull request merged**